src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -40,7 +40,7 @@ public ViewResult StartInfo()
             {
                 return View(new FindProjectViewModel
                 {
-                    OwnerId = (await _userManager.GetUserAsync(User)).Id,
+                    OwnerId = _userManager.GetUserId(User),
```

Comment:
removing "await" means the find page takes a bit more resources.

Have you tried:
* User == null ? null : (await _userManager.GetUserAsync(User)).Id
Or
* (await _userManager.GetUserAsync(User))?.Id